web/public/create_a_game.ejs

```diff
@@ -0,0 +1,33 @@
+<%- include('partials/header') %>
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="style/create_a_game.css">
+</head>
+<body>
+	<main>
+	<div id="game-creation">
+		<div id="question-container">
+			<button id="open-question-modal">></button>
+			<div id="questionModal" class="modal">
+				<div class="modal-content">
```

Comment:
might cause issues for other modals, if they dont use the same display properties.